CORE/Source/Database_Drivers/uRESTDWDriverBase.pas

```diff
@@ -4615,4 +4708,306 @@ procedure TRESTDWDriverBase.BuildReflectionChanges(var ReflectionChanges: String
   End;
 end;
 
-End.
+{ TRDWDrvParams }
+
+function TRDWDrvParams.FindParam(param: string): TRDWDrvParam;
+var
+  I: Integer;
+  prm : TRDWDrvParam;
+begin
+  for I := 0 to Count - 1 do begin
+    prm := Items[i];
+    if SameText(prm.Name,param) then begin
+      Result := prm;
+      Exit;
+    end;
+  end;
+  Result := nil;

```

Comment:
essa inicialização do Result, coloca antes do FOR senão ao terminar o for vai passar por essa linha 4725